web3.js/src/publickey.ts

```diff
@@ -143,10 +143,10 @@ export class PublicKey extends Struct {
    * Derive a program address from seeds and a program ID.
    */
   /* eslint-disable require-await */
```

Comment:
nit: Probably don't need this comment, right?  Makes me wonder if this needed to be async for some reason